### PR TITLE
NOTIF-361 Protect Event log with new RBAC permission

### PR DESCRIPTION
**Pull request by bennetelli**

### CODE REVIEW

**Comment by theute** [comment]
Looks good to me

File: configs/prod/roles/notifications.json [hunk 1]
```diff
@@ -10,6 +10,9 @@
         {
           "permission": "notifications:*:*"
         },
+        {
+          "permission": "events:*:*"
```
Comment (josejulio) [inline configs/prod/roles/notifications.json:14]: same as above

File: configs/qa/roles/notifications.json [hunk 2]
```diff
@@ -10,6 +10,9 @@
         {
           "permission": "notifications:*:*"
         },
+        {
+          "permission": "events:*:*"
```
Comment (josejulio) [inline configs/qa/roles/notifications.json:14]: and the same

File: configs/stage/roles/notifications.json [hunk 3]
```diff
@@ -10,6 +10,9 @@
         {
           "permission": "notifications:*:*"
         },
+        {
+          "permission": "events:*:*"
```
Comment (josejulio) [inline configs/stage/roles/notifications.json:14]: same

File: configs/ci/roles/notifications.json [hunk 4]
```diff
@@ -10,6 +10,9 @@
         {
           "permission": "notifications:*:*"
         },
+        {
+          "permission": "events:*:*"
```
[the remark this answers is not in the document]
Comment (josejulio) [reply configs/ci/roles/notifications.json:14]: Just to clarify, we want `events` to be part of notifications, not an app by itself.

File: configs/stage/roles/notifications.json (hunk 3 above)
Comment (bennetelli) [reply configs/stage/roles/notifications.json:14]: not sure. That was my question as well. Maybe @theute knows if this is needed or not?

Comment (josejulio) [reply configs/stage/roles/notifications.json:14]: I don't think we need it

Comment (theute) [reply configs/stage/roles/notifications.json:14]: @josejulio is right, format is application:<resource type>:<verb>
https://consoledot.pages.redhat.com/docs/dev/services/rbac.html

**Review by astrozzc — APPROVED** [review]

**Comment by josejulio** [comment]
We will need to revert some changes in this PR

**Comment by bennetelli** [comment]
> We will need to revert some changes in this PR

should it be notifications:events:* ? or how should it look like?

**Comment by theute** [comment]
There is already `notifications:*:*` so `notifications:events:*` is already covered.
But  "events:*:*" needs to go away in roles since `events` is not an application

**Comment by astrozzc** [comment]
Oops, the page is not refreshed when I merge it. Just saw the following comments. Will wait for another PR to fix it. @bennetelli Please ping me when you have the fix PR ready

**Comment by bennetelli** [comment]
@astrozzc i will

**Pull request merged**